packages/core/src/common/classes.ts

```diff
@@ -439,3 +440,20 @@ export function positionClass(position: Position | undefined) {
     }
     return `${NS}-position-${position}`;
 }
+
+export function sizeClass(
+    size: Size | Omit<Size, "small">,
+    large: boolean = false,
+    small: boolean = false,
```

Comment:
Having these two boolean optional props next to each other feels ripe for being messed up. I would suggest using an object parameter here:
```suggestion
    legacyProps: Partial<Record<"large" | "small", boolean>>,
```

Reply:
bba8d5aec50ccfb1791117edf6bf63dee97a570c